tests/FSharp.Compiler.Private.Scripting.UnitTests/FSharpScriptTests.fs

```diff
@@ -85,25 +87,6 @@ x
         )
 #endif
 
-    [<Fact>]
```

Comment:
These are moved?

Reply:
Will be rewritten although they make little sense now. They test capture the console streams but probably should deal with the FSI streams which can be different from the console streams.